source/clear-linux/tutorials/smb/smb.rst

```diff
@@ -0,0 +1,101 @@
+.. _clear-samba-share-to-Windows:
+
+Enabling simple file sharing from Clear Linux to a Windows machine using Samba
+##############################################################################
+
+This guide covers the steps you need to enable simple file sharing
+from |CLOSIA| to Windows. For more advanced sharing, please refer to the
+`Samba guide`_.
+
+#.	Follow the :ref:`install-on-target` guide to install |CL|.
+#.	Add the storage-utils bundle, which includes the Samba binaries, after |CL|
+	is setup. The os-clr-on-clr bundle also includes the Samba binaries.
+
+	.. code-block:: console
+
+		$sudo swupd bundle-add storage-utils
+
+#.	Create a configuration file at :file:`/etc/samba/smb.conf` and add the following
+	(change as needed). The following configuration has two simple examples.
```

Comment:
@eadamsintel Can you please update whether this PR is ready to be merged. I found this note an stopped my review.